elucidate-server/src/main/java/com/digirati/elucidate/infrastructure/security/impl/JwtUserSecurityDetailsContext.java

```diff
@@ -20,6 +21,9 @@
     @Override
     public boolean isAuthorized(Permission operation, AbstractAnnotation annotation) {
         Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+        if (auth instanceof AnonymousAuthenticationToken) {
```

Comment:
Hm, the effect of this is that anonymous users are authorized for every action over an annotation. Is this what you're after? I think it's missing an `operation == Permission.READ` check.

Reply:
I was under the impression that the READ part already is taken care of because of the antMatcher only covering GET methods (only the `permitAll()` path creates a `AnonymousAuthenticationToken`):
https://github.com/dlcs/elucidate-server/blob/041ab5ef36fe2936f5b43029487ca99c484f3d5b/elucidate-server/src/main/java/com/digirati/elucidate/infrastructure/config/AuthConfig.java#L135-L141

I'll take a closer look into doing more checking on the operation in `JwtUserSecurityDetailsContext` which would be more robust anyway.

Reply:
Ah, you're right. This is fine as is then. I don't think the test for the anonymous token would be any clearer without representing an anonymous user as a complete `UserDetails` instance.

I'm happy to merge this as is.